composer.json

```diff
@@ -49,7 +49,8 @@
         "sllh/php-cs-fixer-styleci-bridge": "^2.0",
         "sonata-project/intl-bundle": "^2.2.4",
         "symfony/phpunit-bridge": "^2.7 || ^3.0",
-        "symfony/yaml": "^2.3 || ^3.0"
+        "symfony/yaml": "^2.3 || ^3.0",
+        "matthiasnoback/symfony-dependency-injection-test": "^0.7.6"
```

Comment:
https://github.com/sonata-project/SonataAdminBundle/blob/3.x/composer.json#L73

is already done